### PR TITLE
Adds backstories and job descriptions

### PR DESCRIPTION
**Pull request by Kmc2000**
:cl: Kmc2000
add: You can now set a backstory for your character in the setup occupation window
/:cl:

https://i.gyazo.com/2afde3dec65efd58597bee67a813896a.mp4

### CODE REVIEW

File: code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -9,12 +9,17 @@ Captain
...
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "Nanotrasen officials and Space law"
+	supervisors = "Starfleet Admirals and the prime directive."
 	selection_color = "#ccccff"
 	req_admin_notify = 1
 	minimal_player_age = 14
 	exp_requirements = 180
 	exp_type = EXP_TYPE_CREW
+	description = "These are the voyages.<br>The captain of a vessel takes total responsibility for upholding the values of the federation and is responsible for the entire crew, so this decision should not be taken lightly.<br>\
```
Comment (Jalleo) [inline code/modules/jobs/job_types/captain.dm:18]: Too much text here cut it down a bit

File: code/modules/jobs/job_types/job.dm [hunk 2]
```diff
@@ -56,6 +56,8 @@
 	var/paycheck_department = ACCOUNT_CIV
 
 	var/list/mind_traits // Traits added to the mind of the mob assigned this job
+	var/description = "You are a minor crewman aboard a starship, your duties are minimal. <br>After graduating from the academy, all recruits are sent on a 1 year tour in space, are you a fresh academy graduate? or perhaps, you never got the chance for promotion.\
+	If you're inexperienced this is the perfect starter role for you, but there is no restriction on who can play minor roles. Your story is your own, for whatever reason you've arrived aboard a Federation starship with your crewmates. What you do now is up to you."
```
Comment (Jalleo) [inline code/modules/jobs/job_types/job.dm]: Try condensing this area too

File: DS13/code/modules/client/preferences.dm [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+		if("rebel")
+			backstory_reason = "I joined starfleet to find somewhere I belong.<br>"
+		if("insane")
+			backstory_reason = "I joined starfleet to kill hostile aliens.<br>"
+	choice = input(user,"What were you like at starfleet academy?", "Backstory", null) in responses
+	if(!choice)
+		return
+	switch(choice)
+		if("keen")
+			backstory_academy = "At the Academy, I worked hard and received top marks in everything I did. It wasn't easy, but it was worth it. I was put on the fast track to the starfleet command training program.<br>"
```
Comment (Jalleo) [inline DS13/code/modules/client/preferences.dm:42]: change it so backstory_academy already has "At the Academy, " and do a backstory_academy += "I worked hard etc etc" 
Saves slightly repetition.
Saves slightly repetition.

Comment (Kmc2000) [reply DS13/code/modules/client/preferences.dm:42]: initial(academy) + thing

**Review by Jalleo — APPROVED** [review]
Should be fine.

**Pull request merged**